numba/tests/test_random.py

```diff
@@ -61,6 +61,9 @@ def numpy_multinomial2(n, pvals):
 def numpy_multinomial3(n, pvals, size):
     return np.random.multinomial(n, pvals=pvals, size=size)
 
+def numpy_dirichlet(alpha, size):
+    return np.random.dirichlet(alpha, size=size)
```

Comment:
A second variant of this, like:
```python
def numpy_dirichlet(alpha):
    return np.random.dirichlet(alpha)
```
should be tested to ensure that default arguments work as anticipated.

Reply:
Added this in.